### PR TITLE
feat: cache metrics

### PR DESCRIPTION
**Pull request by simone-sanfratello**
Add both metrics for blocks data and info

### CODE REVIEW

File: test/health-check.test.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 'use strict'
 
 process.env.CACHE_BLOCKS_INFO = 'true'
+process.env.CACHE_BLOCKS_DATA = 'true'
```
Comment (alanshaw) [inline test/health-check.test.js:4]: The actual env var name used is `CACHE_BLOCK_DATA` (no `S`) so this won't have any effect atm.

**Review by alanshaw — APPROVED** [review]
LGTM, just a minor typo in a few places that needs to be cleaned up 😄

**Comment by simone-sanfratello** [comment]
@francardoso93 we probably need to update those env vars on infra too

File: src/config.js [hunk 2]
```diff
@@ -41,8 +41,8 @@ const httpPort = parseInt(rawHttpPort)
 
 module.exports = {
   blocksTable: blocksTable ?? 'blocks',
-  cacheBlocksInfo: cacheBlocksInfo === 'true',
-  cacheBlocksSize: cacheBlocksSize ?? 1e3,
+  cacheBlockInfo: cacheBlockInfo === 'true',
+  cacheBlockInfoSize: cacheBlockInfoSize ?? 1e3,
```
Comment (alanshaw) [inline src/config.js:45]: You should `parseInt` on `cacheBlockInfoSize` because it is a string if set.

File: src/service.js [hunk 3]
```diff
@@ -25,7 +25,7 @@ const {
 const { cidToKey, defaultDispatcher, fetchBlockFromS3, searchCarInDynamoV1 } = require('./storage')
 const { telemetry } = require('./telemetry')
 
-const blocksCache = new LRUCache(cacheBlocksSize)
+const blocksInfoCache = new LRUCache(cacheBlockInfoSize)
```
Comment (alanshaw) [inline src/service.js:28]:
```suggestion
const blockInfoCache = new LRUCache(cacheBlockInfoSize)
```

File: src/service.js [hunk 4]
```diff
@@ -34,10 +34,14 @@ function createEmptyMessage(blocks = [], presences = []) {
 
 async function getBlockInfo(dispatcher, cid) {
   const key = cidToKey(cid)
-  const cached = blocksCache.get(key)
+  const cached = blocksInfoCache.get(key)
```
Comment (alanshaw) [inline src/service.js:37]:
```suggestion
  const cached = blockInfoCache.get(key)
```

File: src/service.js [hunk 5]
```diff
@@ -47,7 +51,7 @@ async function getBlockInfo(dispatcher, cid) {
   )
 
   if (item) {
-    blocksCache.set(key, item)
+    blocksInfoCache.set(key, item)
```
Comment (alanshaw) [inline src/service.js:54]:
```suggestion
    blockInfoCache.set(key, item)
```

**Review by alanshaw — CHANGES_REQUESTED** [review]

**Review by alanshaw — APPROVED** [review]
🚀 beautiful

**Pull request merged**